code/controllers/subsystem/ticker.dm

```diff
@@ -43,7 +43,8 @@ SUBSYSTEM_DEF(ticker)
 	var/timeLeft						//pregame timer
 	var/start_at
 
-	var/gametime_offset = 432000 // equal to 12 hours, making gametime at roundstart 12:00:00
+	var/gametime_offset = 432000		//Deciseconds to add to world.time for station time.
+	var/station_time_rate_multiplier = 24		//factor of station time progressal vs real time.
```

Comment:
Remind me to override this in modular_citadel to be 1:1 with realworld time to preserve the current nightshift behavior

Reply:
her nerd, override this in modular_citadel. 

c:

Reply:
also tbh, I like it randomized, give the daywalkers a chance to enjoy moody light